CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ include(CTest)
 set(CMAKE_CXX_STANDARD 20)
 set(CXX_STANDARD_REQUIRED ON)
 set(CMAKE_POLICY_DEFAULT_CMP0063 NEW) # Set CMAKE visibility policy to NEW on project and subprojects
-set(CMAKE_CXX_VISIBILITY_PRESET hidden) # Set C++ symbol visibility to default to hidden
+# set(CMAKE_CXX_VISIBILITY_PRESET hidden) # Set C++ symbol visibility to default to hidden
```

Comment:
Why is this commented out? If this is affecting compilation, it's likely to be a cross-platform issue given MSVC's default symbol visibility.

Reply:
These two lines are removed in another PR.

Reply:
This should not be part of this PR then.

Reply:
Reverted